static/SimpleInventory/js/edit-board.js

```diff
@@ -5,11 +5,11 @@ function redirect(event) {
   event.preventDefault();
   var form_data = $('#editBoardForm');
     $.post(event.target.action, form_data.serialize(), function(response) {
-	window.location.assign("/view/list");
```

Comment:
This cannot be a hardcoded route (not to mention this still doesn't work even as a hardcoded route).  You need to get the route by name from Django.